queries.sql

```diff
@@ -131,17 +131,65 @@ JOIN owners ON owner_id = owners.id
 WHERE owners.full_name = 'Dean Winchester' AND animals.escape_attempts = 0;
 
 /*Who owns the most animals?*/
-SELECT full_name, COUNT(animals.name) as animal_count
-FROM owners
+SELECT full_name, COUNT(animals.name) as total FROM owners
 JOIN animals ON animals.owner_id = owners.id
 GROUP BY full_name
-ORDER BY animal_count DESC LIMIT 1;
-
-
-
-
-
+ORDER BY total DESC LIMIT 1;
 
+/*-- Who was the last animal seen by William Tatcher?*/
+SELECT animals.name, visits.date_of_visits
+FROM animals
+JOIN visits ON animals.id = visits.animal_id
+JOIN vets ON vets.id = visits.vet_id
+WHERE vets.name = 'William Tatcher'
+ORDER BY visits.date_of_visits DESC
+LIMIT 1;
+
```

Comment:
- [ ] Please add the query seen in the screenshot below and its screenshot. You missed it :eyes: 
![image](https://github.com/wolterbwambale/vet-clinic/assets/45460192/26875279-7d09-46dc-87bb-669473e4b331)